kie-dmn-validation/src/main/resources/rules.drl

```diff
@@ -201,6 +203,8 @@ rule NAME_INVALID
 when
     $ne : NamedElement( FEELParser.isVariableNameValid( name ) == false )
 then
+//    java.util.List<FEELEvent> see = FEELParser.checkVariableName( $ne.getName() );
+//    String customMessage = see.isEmpty()? Msg.NAME_INVALID : see.get(0).getMessage()
     entryPoints["PROBLEMS"].insert( new ValidationMsg( Severity.ERROR, Msg.NAME_INVALID, $ne ) );
```

Comment:
@tarilabs I am afraid we will need to convert the Msg enum into a class like you did for the FEEL module, otherwise we can't create customized messages like the one here for the invalid variable name? The commented lines above show how to get a proper error message in case the variable name is invalid.